plugin-packs/postcss-preset-env/src/types/cssdb/index.d.ts

```diff
@@ -0,0 +1,17 @@
+declare module 'cssdb' {
+	export type cssdbEntry = {
+		id: string
+		title: string
+		description: string
+		specification: string
+		stage: number
+		browser_support: Record<string, string>
+		docs: Record<string, string>
+		example: string
+		polyfills: Array<Record<string, string>>,
+		vendors_implementations: number
+	};
+
+	const cssdb: Array<cssdbEntry>;
+	export default cssdb;
+}
```

Comment:
Got a nag because `cssdb` isn't typed.
We don't use typescript for the internals of `postcss-preset-env` so these might not be complete, but currently this is fine.